app/serializers/admin_plugin_serializer.rb

```diff
@@ -10,7 +10,9 @@ class AdminPluginSerializer < ApplicationSerializer
              :enabled,
              :enabled_setting,
              :has_settings,
-             :is_official
+             :is_official,
+             :long_commit_hash,
+             :short_commit_hash
```

Comment:
Is there a need to ship down both long and short commit hash from the server when the short commit hash can easily be derived from the long commit hash on the client side?